### DISCUSSION TITLE
What identified the predicates in the role labeling demo?

### DISCUSSION THREAD

**Discussion by RevanthRameshkumar** [allenai/allennlp#5704] [Q&A]
I read this paper:
```
Simple BERT Models for Relation Extraction and Semantic Role Labeling
https://arxiv.org/pdf/1904.05255.pdf
```
and it mentions for the role labeling, the predicate is given for training and test...
yet when I use this example for the demo:
```
I glorpfloped the oven.
```
it correctly identifies glorpfloped as the frame for which arg0 and arg1 are assigned. What system handles the frame/predicate identification here?

**Comment by RevanthRameshkumar** [comment]
After some more searching I think it might be a spacy tagger? Would be great if this can be confirmed

**Reply by dirkgr** [reply]
I am not very familiar with that work, but isn't that just the output of the model? The system that identifies frames is the very model that this work is about?
